src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int longSide;
```

Comment:
Optional to think about: In my mind Triangle and RightTriangle have a kind of relation, maybe it makes sence to create inheritance between them